core/tests/visits.py

```diff
@@ -77,6 +80,86 @@ def test_update_visit_notes(self):
         self.assertEqual(Visit.objects.count(), 1)
         self.assertEqual(Visit.objects.get(id=visit_id).notes, new_note)
 
+    def test_update_visit_urgency(self):
+        """
+        Ensure we can update notes on a visit
+        """
+        # create a visit
+        headers = self.auth_headers_for_user("front_desk")
+        data = {
+            "participant": 1,
+            "program": 1,
+            "service": 1,
+            "urgency": "_2",
+        }
+        create_response = self.client.post(
+            "/api/visits/", data, format="json", **headers
+        )
+        self.assertEqual(create_response.status_code, status.HTTP_201_CREATED)
+
+        headers = self.auth_headers_for_user("case_manager")
+        new_urgency = "_3"
+
+        visit_id = json.loads(create_response.content)["id"]
+
+        data = {
+            "id": visit_id, 
+            "urgency": new_urgency,
+        }
+        update_response = self.client.patch(
+            f"/api/visits/{visit_id}/", data, format="json", **headers
+        )
+
+        self.assertEqual(update_response.status_code, status.HTTP_200_OK)
+        self.assertEqual(Visit.objects.count(), 1)
+        self.assertEqual(Visit.objects.get(id=visit_id).urgency, new_urgency)
+
+    def test_update_visit_program_service(self):
+        """
+        Ensure we can update notes on a visit
+        """
+        # create a visit
+        headers = self.auth_headers_for_user("front_desk")
+        data = {
+            "participant": 1,
+            "program": 1,
+            "service": 1,
+            "urgency": "_2",
+        }
+        create_response = self.client.post(
+            "/api/visits/", data, format="json", **headers
+        )
+        self.assertEqual(create_response.status_code, status.HTTP_201_CREATED)
+
+        headers = self.auth_headers_for_user("case_manager")
+        new_note = "I forgot to add notes the first time!"
```

Comment:
doesnt look like this is being used in this test, can you remove?

Reply:
Done